src/style.scss

```diff
@@ -0,0 +1,132 @@
+$bodyFont: "Roboto", sans-serif;
+$productCardMarginBottom: 16px;
+$mainColor: #060b35;
+$secondaryColor: #616070;
+
+body {
+  font-family: $bodyFont;
+  margin: 0;
+}
+
+.product-card {
+  align-items: center;
+  justify-content: center;
+  width: 200px;
+  height: 406px;
+  margin: auto;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+.product-card__photo {
+  margin: 32px 19px 40px;
+  height: 134px;
+  width: 160px;
+  align-items: center;
+  justify-content: center;
+
+  .product-card__img {
+    width: 160px;
+    height: 134px;
+  }
+}
+
+.product-card__article {
+  margin: 0 $productCardMarginBottom 4px $productCardMarginBottom;
```

Comment:
if your variable called marginBottom, why do you use it for margin left and right?